### PR TITLE
Add: working incompatibility fix

### PR DESCRIPTION
**Pull request by dalisoft**
Related issue: #140, [here](https://github.com/uNetworking/uWebSockets.js/issues/158) or [here](https://github.com/dalisoft/nanoexpress/issues/6)

### CODE REVIEW

**Comment by mhart** [comment]
Oh wow – that seems like a pretty extreme solution – I'd be worried a lot of system software might break with this change?

**Comment by dalisoft** [comment]
No, it just adds missing file. That's all, i checked a lot of times. Also spent almost 20 hour to find fix for this :)

**Comment by mhart** [comment]
Ohhh, I see – sorry, didn't look at what you were doing correctly, I thought you were overriding the musl library with a softlink to libc – instead of the other way around 😄

**Comment by dalisoft** [comment]
No prob, thanks for amazing small Docker Node.js image, works very fast and best

File: README.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ Alpine is probably the easiest way to do this).
 
 If you get an error similar to `error loading shared library ld-linux-x86-64.so.2`, it may be that you have dependencies relying on libc – you can try to fix this by adding `apk add --no-cache libc6-compat` to your `Dockerfile`.
 
+If adding `libc6-compat` doesn't help, you can try adding `RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2` before running your `node` app.
```
Comment (mhart) [inline README.md]: I'd actually just make it part of the last paragraph. Reword it to:

```
you can try to fix this by adding `RUN apk add --no-cache libc6-compat` or `RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2` to your `Dockerfile`.
```

Comment (dalisoft) [reply README.md]: Ok, give me minute :)

Comment (dalisoft) [reply README.md]: I fixed :)

**Comment by mhart** [comment]
Thanks so much! Merging 👍

**Comment by dalisoft** [comment]
Thank you for merge =)
